host/sgx/windows/host_context.asm

```diff
@@ -0,0 +1,78 @@
+;; Copyright (c) Microsoft Corporation. All rights reserved.
+;; Licensed under the MIT License.
+
+OE_CONTEXT_FLAGS    EQU 00
+OE_CONTEXT_RBX      EQU 08
+OE_CONTEXT_RDI      EQU 16
+OE_CONTEXT_RSI      EQU 24
+OE_CONTEXT_R12      EQU 32
+OE_CONTEXT_R13      EQU 40
+OE_CONTEXT_R14      EQU 48
+OE_CONTEXT_R15      EQU 56
+OE_CONTEXT_FLOAT    EQU 64
+
+.CODE
+
+PUBLIC oe_save_host_context
+oe_save_host_context PROC
+    ;; Subroutine prologue
+    push rbp 
+    mov rbp, rsp
+
+    ;; Save the flags to stack.
+    pushf
```

Comment:
Should this function also save MXCSR?

Reply:
MXCSR is preserved as part of the [fxsave instruction](https://www.felixcloutier.com/x86/fxsave#tbl-3-43), see comment on line 37

Reply:
Great.